Doc generation fails with the Doxygen version from meta-openembedded (which
is newer than the one in meta-ti). This should be reported to TI if it is
still an issue with newer Yocto releases.

See also: https://e2e.ti.com/support/processors-group/processors/f/processors-forum/770498/linux-am5706-compilation-failure-in-meta-ti-common-csl-ip-rtos-recipe

--- a/docs/doxyfile.xdt
+++ b/docs/doxyfile.xdt
@@ -134,7 +134,7 @@ HTML_HEADER            = ./docs/tiheader.htm
 HTML_FOOTER            = ./docs/tifooter.htm
 HTML_STYLESHEET        = 
 HTML_ALIGN_MEMBERS     = YES
-GENERATE_HTMLHELP      = YES
+GENERATE_HTMLHELP      = NO
 CHM_FILE               = ..\..\csldocs.chm
 HHC_LOCATION           =
 GENERATE_CHI           = NO
